django_tablib/base.py

```diff
@@ -81,3 +84,14 @@ def append(self, *args, **kwargs):
             row = django_object
 
         super(BaseDataset, self).append(row=row, col=col)
+
+
+def resolve_related_value(obj, attr):
+    fields = attr.split('.')
+    related_field = obj
+    for field in fields:
+        related_field = getattr(related_field, field)
+        if related_field is None:
```

Comment:
It took me a moment to understand this. Not a blocker, but it might be helpful to comment e.g. 
```# If FK is null, stop following the attr chain and return None```